src/apps/Hosting/HostingDialog.js

```diff
@@ -124,16 +148,21 @@ const CreateHostingDialog = React.createClass({
             floatingLabelText="Description"
             data-e2e="hosting-dialog-description-input"
           />
-          <SelectWrapper
-            errorText={this.getValidationMessages('domains').join(' ')}
-            className="vm-5-t"
-            multi={true}
-            value={domains}
-            allowCreate={true}
-            placeholder="Domains"
-            onChange={this.handleChangeDomain}
-            data-e2e="hosting-dialog-domains-input"
+          <HostingDialogDomainTable
+            domains={domains}
+            newDomain={newDomain}
+            handleChangeNewDomain={this.handleChangeNewDomain}
+            handleAddNewDomain={this.handleAddNewDomain}
+            handleChangeDomains={this.handleChangeDomains}
+            handleRemoveDomain={this.handleRemoveDomain}
           />
+          <div className="vm-2-t">
```

Comment:
you can add `className` directly to the Notification component. it won't keep that margin when it's closed.